OrcScala/src/orc/values/sites/SiteResolver.scala

```diff
@@ -18,7 +18,7 @@ package orc.values.sites
   * @author jthywiss
   */
 trait SiteResolver {
-  def resolve(name: String): Site
+  def resolve(name: String): AnyRef
```

Comment:
Because Java interop "sites" do not have type Site any more. They are just Class[_] instances (for classes) or any Java object. The dispatch is handled by generating invokers without wrapping.

SiteResolver might be a misnomer now. Though in some sense it's still accurate since those Java objects are created as sites.